packages/bot-web-ui/src/stores/toolbar-store.js

```diff
@@ -37,7 +37,6 @@ export default class ToolbarStore {
                 const scroll_distance = toolbox_width - block_canvas_rect.left + toolbox.width;
                 scrollWorkspace(workspace, scroll_distance, true, false);
             }
-            this.root_store.core.gtm.pushDataLayer({ event: 'dbot_toolbox_visible', value: true });
```

Comment:
Was this removed for any reason?

Reply:
I think I accidentally remove it. THank you 🙏